docs/feature_flags.md

```diff
@@ -94,3 +94,10 @@ computed at all.
 `--enable-feature=auto-gomaxprocs`
 
 When enabled, GOMAXPROCS variable is automatically set to match Linux container CPU quota.
+
+## No Default Scrape Port
+
+`--enable-feature=no-default-scrape-port`
+
+When enabled, the default port for HTTP or HTTPS won't be added a target's address if no port is specified.
+In addition, if a default port (`:80` or `:443`) has already been attached, it will be removed.
```

Comment:
s/won't be added a/won't be added to a/

Could you add some detail on where this takes effect?  
I see the main change is in `scrape.PopulateLabels` - does it only impact labels?
Does it affect the address sent in the http request?
Who would have "already attached" a port?

Reply:
Thanks for pointing that out. I took a shot at writing a clearer and more comprehensive description.